hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/CallOutcomeTest.java

```diff
@@ -40,25 +41,20 @@ class CallOutcomeTest {
     @Test
     void recognizesCreatedIdWhenEvmAddressIsSet() {
         given(updater.getCreatedContractIds()).willReturn(List.of(CALLED_CONTRACT_ID));
-        final var outcome = new CallOutcome(TestHelpers.SUCCESS_RESULT.asProtoResultOf(updater), SUCCESS);
+        final var outcome = new CallOutcome(SUCCESS_RESULT.asProtoResultOf(updater), SUCCESS, NETWORK_GAS_PRICE);
         assertEquals(CALLED_CONTRACT_ID, outcome.recipientIdIfCreated());
     }
 
     @Test
     void recognizesNoCreatedIdWhenEvmAddressNotSet() {
-        final var outcome = new CallOutcome(TestHelpers.SUCCESS_RESULT.asProtoResultOf(updater), SUCCESS);
+        final var outcome = new CallOutcome(SUCCESS_RESULT.asProtoResultOf(updater), SUCCESS, NETWORK_GAS_PRICE);
         assertNull(outcome.recipientIdIfCreated());
     }
 
-    @Test
-    void calledIdIsNullIfNoResult() {
-        final var outcome = new CallOutcome(null, INVALID_CONTRACT_ID);
-        assertNull(outcome.recipientIdIfCalled());
-    }
-
     @Test
     void calledIdIsFromResultIfExtant() {
```

Comment:
Is `...IfExtant` still meaningful?

Reply:
Not any more, I'm not sure what condition I thought `CallOutcome#result` would be null but it's gone afaict.